src/Estimators/CSEnergyEstimator.h

```diff
@@ -46,12 +48,15 @@ struct CSEnergyEstimator : public ScalarEstimatorBase
   Matrix<RealType> tmp_data;
   ///name of hamiltonian components
   std::vector<std::string> h_components;
+  CSLocalEnergyInput input_;
```

Comment:
Is it feasible to make all the input_ const? Thinking of just move in during constructor and then const forever.

Reply:
Looks like this is doable.